### PR TITLE
Operation docs

### PR DESCRIPTION
**Pull request by N-J-Martin**
notebooks and bits files for the various operators and toInt.

### CODE REVIEW

File: docs/notebooks/addition_subtraction_examples.ipynb
Comment (hz66-404) [inline docs/notebooks/addition_subtraction_examples.ipynb:1]: add colab title to this doc will be better

File: docs/notebooks/absoluteValues.ipynb
Comment (hz66-404) [inline docs/notebooks/absoluteValues.ipynb:1]: the fourth cell has error

File: docs/notebooks/factorial.ipynb
Comment (hz66-404) [inline docs/notebooks/factorial.ipynb:1]: the second cell has error

File: docs/notebooks/addition_subtraction_examples.ipynb
Comment (N-J-Martin) [reply docs/notebooks/addition_subtraction_examples.ipynb:1]: done!

File: docs/notebooks/factorial.ipynb
Comment (N-J-Martin) [reply docs/notebooks/factorial.ipynb:1]: there's is supposed to be an error there. It is demonstrating that it won't work the way you expect.

File: docs/notebooks/absoluteValues.ipynb
Comment (N-J-Martin) [reply docs/notebooks/absoluteValues.ipynb:1]: made it clearer that it is meant to be demonstrating the error.

**Pull request merged**